test/unit-test/data.tf

```diff
@@ -21,6 +21,7 @@ data "aws_iam_policy_document" "ec2_test_policy" {
     #checkov:skip=CKV_AWS_109:
     #checkov:skip=CKV_AWS_111:
     #checkov:skip=CKV_AWS_107:
+    #checkov:skip=CKV_AWS_356:
```

Comment:
Could you please add a description with a reason for skipping?

Reply:
I was being consistent with the previous skips :). It's because the policy is just used within the unit-test only

Reply:
I know, we've been lazy before :) but good to have that extra info going forward :)